spring/src/main/java/org/axonframework/spring/authorization/UnauthorizedCommandException.java

```diff
@@ -0,0 +1,18 @@
+package org.axonframework.spring.authorization;
+
+/**
+ * Exception indicating that a command has been rejected due to a lack of authorization.
+ *
+ * @author Roald Bankras
+ */
+public class UnauthorizedCommandException extends RuntimeException {
```

Comment:
If we'd make the interceptors more generic, I think we can rename the exception to `UnauthorizedMessageException`.